### PR TITLE
ComposeViewController with Texture

### LINKED ISSUE #58: migrate to Texture, declare ui in code
My biggest worry is that if we do all UI in code, there will be too much code, or code readability will suffer by mixing UI and logic.

Hopefully you can find a way to minimize that.

### PR DESCRIPTION
**Pull request by Kharchevskyi**
close #58

### CODE REVIEW

**Comment by Kharchevskyi** [comment]
Simulator Screen Shot - iPhone 8 - 2019-11-06 at 19 16 16

File: FlowCrypt/Common/Services/StorageService.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    func publicKey() -> String?
+    func keys() -> Results<KeyInfo>
+}
+
+struct StorageService: StorageServiceType {
+    var storage: Realm {
+        return try! Realm()
+    }
+
+    func store(keyDetails: [KeyDetails], passPhrase: String, source: KeySource) {
```
Comment (tomholub) [inline FlowCrypt/Common/Services/StorageService.swift]: Rename to `addKeys`

File: FlowCrypt/Controllers/Compose/ComposeViewController.swift [hunk 2]
```diff
@@ -138,55 +118,68 @@ extension ComposeViewController {
...
+            let pgpText = "compose_missed_public".localized
+
             let lookupRes = try await(self.attesterApi.lookupEmail(email: email))
-            guard let recipientPubkey = lookupRes.armored else { return self.showAlert(message: pgpText) }
-            let realm = try Realm() // TODO: Anton - Refactor to use db service
 
-            guard let myPubkey = realm.objects(KeyInfo.self).map({ $0.public }).first else { return self.showAlert(message: pgpText) }
-            let encrypted = self.encryptMsg(pubkeys: [myPubkey, recipientPubkey], subject: subject, message: text, email: email)
+            guard let recipientPubkey = lookupRes.armored else {
+                return self.showAlert(message: pgpText)
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeViewController.swift]: maybe can directly put `"compose_missed_public".localized`, it's more expressive

File: FlowCrypt/Controllers/Compose/ComposeViewController.swift [hunk 3]
```diff
@@ -138,55 +118,68 @@ extension ComposeViewController {
...
+            }
+
+            guard let myPubkey = self.storageService.publicKey() else {
+                return self.showAlert(message: pgpText)
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeViewController.swift]: alert: `Could not find any suitable Public Key on your account. If FlowCrypt app well set up?`

**Comment by tomholub** [comment]
I will rename some things and do some testing, else it looks good!

**Pull request merged**